test/Project.toml

```
@@ -17,6 +17,7 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 Aqua = "0.7"
 BSON = "0.3.3"
 CairoMakie = "0.6, 0.7, 0.8, 0.9, 0.10"
+Downloads = "1"
```

```suggestion
Downloads = "1.8"
```
It should have the same version number as the Julia entry in our main `Project.toml` - see https://discourse.julialang.org/t/psa-compat-requirements-in-the-general-registry-are-changing/104958